### ISSUE TITLE
Write a log when backends are detected down or up.

### ISSUE THREAD

**Issue by FlorentinDUBOIS** [sozu-proxy/sozu#892]
Hello,

I was talking about Sōzu internally at Clever Cloud. 
We found out that for debugging purpose, it will be helpful like haproxy already does, to write a log when a backend of a cluster is down or up.
We could also write a log when all backends of a cluster are down.

Have a nice day.

**Comment by StevenLeRoux** [comment]
I agree and I advise to also expose a metric of a counter for available targets per backend

**Comment by FlorentinDUBOIS** [comment]
Suggested by @BlackYoup, we could also log into access logs timeouts that occurs on requests.
